package/packer/gcp/template.json

```diff
@@ -0,0 +1,54 @@
+{
+  "variables": {
+    "gcp_project_id": "{{env `GCP_PROJECT_ID`}}",
+    "region": "us-east1-d",
+    "k3os_version": "v0.3.0",
+    "iso_url": "https://github.com/rancher/k3os/releases/download/v0.3.0/k3os-amd64.iso"
+  },
+  "builders": [
+    {
+      "type": "googlecompute",
+      "account_file": "account.json",
+      "project_id": "{{user `gcp_project_id`}}",
+      "source_image": "ubuntu-1804-bionic-v20190911",
+      "zone": "us-east1-d",
```

Comment:
Can `zone` be parameterized with the `region` var?

Reply:
Of course. Done.